Cache/Hashing.lean

```diff
@@ -14,9 +14,9 @@ open System IO
 
 structure HashMemo where
   rootHash : UInt64
-  depsMap  : Std.HashMap FilePath (Array FilePath) := {}
-  cache    : Std.HashMap FilePath (Option UInt64) := {}
-  hashMap  : HashMap := {}
+  depsMap  : Std.HashMap FilePath (Array FilePath) := ∅
```

Comment:
You're using this as \emptyset is a nicer syntax for an empty set. Fair point, that's something I can get by with.